packages/runtime/client/rest/src/contract/rolesConfigurations.ts

```diff
@@ -0,0 +1,240 @@
+import * as t from "io-ts/lib/index.js";
+
+import { PolicyId, RoleName } from "@marlowe.io/language-core-v1";
+
+import * as G from "@marlowe.io/language-core-v1/guards";
+
+/**
+ *  @category Roles Configuration
+ */
+export interface AddressBech32Brand {
+  readonly AddressBech32: unique symbol;
+}
+
+export const AddressBech32Guard = t.brand(
+  t.string,
+  (s): s is t.Branded<string, AddressBech32Brand> => true,
+  "AddressBech32"
+);
+
+/**
+ *  Cardano Address in a Bech32 format
+ *  @category Roles Configuration
+ */
+export type AddressBech32 = t.TypeOf<typeof AddressBech32Guard>;
```

Comment:
I think we created this in a pair session, but the idea is to replace the runtime-core one with this one and not to create yet another AddressBech32 type

Reply:
https://github.com/input-output-hk/marlowe-ts-sdk/issues/117